src/components/GlobalPreferences/GlobalPreferencesButton.js

```diff
@@ -84,17 +85,33 @@ function GlobalPreferencesButton({ onOpen }) {
           </li>
           <Item
             onClick={handleItemClick('network')}
+            iconFirst
             icon={iconNetwork}
             label="Network"
           />
+          <Item
+            iconFirst={false}
+            icon="IconExternal"
+            label="Any questions? Visit our FAQ"
+            href="https://help.aragon.org/article/48-aragon-court-faq"
+          />
         </ul>
       </Popover>
     </React.Fragment>
   )
 }
 
-function Item({ icon, label, onClick }) {
+function Item({ icon, iconFirst, label, onClick, href }) {
```

Comment:
Rather than exposing an API with `iconFirst`, what do you think about exposing `children` instead?

We've used this pattern a few times; with that you can opt for the "easy" pre-defined layout, or customize the internals completely by using children.